extensions/Worker.Extensions.Storage.Blobs/src/Worker.Extensions.Storage.Blobs.csproj

```diff
@@ -14,15 +14,12 @@
 
   <Import Project="..\..\..\build\Extensions.props" />
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
-    <ProjectReference Include="..\..\..\src\DotNetWorker.Core\DotNetWorker.Core.csproj" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" Version="12.19.1" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Azure" Version="1.7.4" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Core" Version="1.19.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.3.0" />
```

Comment:
It looks like in other extensions (except Tables), Abstractions was kept as a project reference. Should this as well? Or should all of them become package references like this one?

Reply:
Ultimately, I would like to see all extensions referencing Abstractions as a package reference, as they should eventually be moved to the appropriate repos and managed independently. I'll make sure those are consistent across extensions, but until we move forward with the move, using a project reference would make sense and avoid confusion.

Reply:
Consolidated them all